scrapyd_k8s/joblogs/log_handler_k8s.py

```diff
@@ -8,12 +8,20 @@
 
 logger = logging.getLogger(__name__)
 
+# Custom Exceptions
```

Comment:
I'd move these custom exceptions to a separate file.

Reply:
Good point! If I make a file with custom exceptions within joblogs module, is it fine? I just feel like making a general file for custom exceptions in the root directory for this optional feature is not a good idea. What do you think?